config/src/parameters/actual.rs

```diff
@@ -72,6 +72,7 @@ pub struct Common {
     pub chain_id: ChainId,
     pub key_pair: KeyPair,
     pub p2p_address: SocketAddr,
+    pub idle_timeout: Duration,
```

Comment:
Maybe group the `p2p_address` with `idle_timeout` into a common struct? Could name it something like `P2p`, `NetworkLayer` or just `Network`